File: 1426-find-n-unique-integers-sum-up-to-zero/find-n-unique-integers-sum-up-to-zero.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        t=int(n/2)
        res=[]
        for i in range(1,t+1):
            res.append(i)
            res.append(-i)
        if n%2!=0:
            res.append(0)
        return res

File: 1426-find-n-unique-integers-sum-up-to-zero/find-n-unique-integers-sum-up-to-zero.java
class Solution {
    public int[] sumZero(int n) {
        int t=n/2;
        int[] res = new int[n];
        int index=0;
        for(int i=1;i<=t;i++){
            res[index++]=i;
            res[index++]=-i;
        }
        if(n%2!=0){
            res[index]=0;
        }
        return res;
    }
}